LayoutTests/model-element/model-element-environment-map-ready.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ ModelElementEnabled=true ModelProcessEnabled=true ] -->
+<meta charset="utf-8">
+<title>&lt;model> ready promise</title>
```

Comment:
Nit: can update the title to mention "environment map"

Reply:
Done.